aws-iot-scheduledaudit/inputs/inputs_1_invalid.json

```diff
@@ -0,0 +1,15 @@
+{
+  "ScheduledAuditName": "CfnContractTest",
```

Comment:
Does the `contract_update_create_only_property` contract test still pass for you on cfn v0.1.14? I had to change this name to be different from the one in create input.